pyanaconda/modules/network/network.py

```diff
@@ -225,15 +256,21 @@ def _hostname_service_properties_changed(self, interface, changed, invalid):
             log.debug("Current hostname changed to %s", hostname)
 
     def get_current_hostname(self):
-        """Return current hostname of the system."""
+        """Return current hostname of the system.
+
+        It will activate the hostname service if it is deactivated.
```

Comment:
It might be useful to hint at the mechanism that does the activation, I am not able to tell how this happens...

(Same for the 3 other docstrings that mention it.)